rosdep/python.yaml

```diff
@@ -6768,6 +6768,62 @@ python3-gnupg:
   openembedded: [python3-gnupg@meta-python]
   opensuse: [python3-python-gnupg]
   ubuntu: [python3-gnupg]
+python3-google-auth:
+  debian:
+    '*': [python3-google-auth]
+    buster:
+      pip:
+        packages: [google-auth]
```

Comment:
I see `python3-google-auth`: https://packages.debian.org/buster/python3-google-auth, so no need for `pip` for `buster`.

Reply:
See https://github.com/ros/rosdistro/pull/33017#discussion_r884109742

Reply:
Sorry for being late, I fixed this part. In addition, I don't use both pip and non-pip keys at the same time as @tfoote 's comment